docs/getting-started.md

```diff
@@ -76,6 +76,16 @@ An Azure Service Principal is needed for populating the controller manifests. Th
   ```
 <!--An alternative is to install [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) and have the project's script create the service principal automatically. _Note that the service principals created by the scripts will not be deleted automatically._ -->
 
+  5. SSH keys
+
+  You can use your own ssh key for testing instead of the test generate a new one for every test. To setup that you need to set the `AZURE_SSH_PUBLIC_KEY_FILE` enviroment variable.
```

Comment:
Probably the language can be simplified here to

 By default a new ssh key is generated, to setup your own ssh key, set the `AZURE_SSH_PUBLIC_KEY_FILE` enviroment variable
    
```bash
export AZURE_SSH_PUBLIC_KEY_FILE=<PATH_FOR_PUBLIC_SSH_KEY>
```

Reply:
done